CONTRIBUTING.md

```diff
@@ -1,15 +1,143 @@
 # Contributing to kubernetes/test-infra
 
-Make a [pull request](https://help.github.com/articles/about-pull-requests/) (hereafter PR). 
-In the PR body, feel free to add an area label if appropriate by saying `/area <AREA>`. 
-The list of labels is [here](https://github.com/kubernetes/test-infra/labels). 
-Also feel free to suggest a reviewer with `/assign @theirname`.
-
-Once your reviewer is happy, they will say `/lgtm` which will apply the 
-`lgtm` label, and will apply the `approved` label if they are an 
-[owner](/OWNERS).
-The `approved` label will also automatically be applied to PRs opened by an 
-OWNER. If neither you nor your reviewer is an owner, please `/assign` someone
- who is.
-Your PR will be automatically merged once it has the `lgtm` and `approved` 
-labels, does not have any `do-not-merge/*` labels, and all tests are passing.
+Welcome! If you haven't seen them already, the Kubernetes project has:
+
+- A [Contributor Guide][contrib-guide] - some [kubernetes/kubernetes]-specific content, but lots of info for the entire project
+- A [Contributor Cheat Sheet][contrib-cheatsheet] - lots of resources and handy links
+
+## Contact
+
+This repo is owned by [SIG Testing][sig-testing], and the people most directly
+involved are listed in [/OWNERS]. We are most responsive on our slack channel
+
+- Community Site: [kubernetes/community/sig-testing][sig-testing]
+- Slack: [#sig-testing]
+- Mailing List: [kubernetes-sig-testing@]
+
+We also have two other slack channels:
+
+- [#testing-ops]: for [prow.k8s.io] and other test-infra ops. Ping `@test-infra-oncall` or check [go.k8s.io/oncall]
+- [#prow]: for discussions related to the use or development of [`prow`](/prow)
+
+## Workflow
+
+In general we use the same pull request (PR) and review workflow used by the
+rest of the Kubernetes project. This means the [use of a CLA][cla], [code
```

Comment:
Maybe `s/use of a CLA/use of the CNCF CLA/`, it's only slightly longer and if I need to sign a CLA, I want to know whose. Ideally I don't have to click a link to find out.

Reply:
done